src/comm/TcpAcceptor.cc

```diff
@@ -364,6 +364,9 @@ Comm::TcpAcceptor::oldAccept(Comm::ConnectionPointer &details)
         if (ignoreErrno(errcode)) {
```

Comment:
A system call error that Squid has no control over and can safely ignore is usually not `DBG_IMPORTANT`. Please consider merging the new check with the existing ignoreErrno case instead:

```suggestion
         if (ignoreErrno(errcode) || errcode == ECONNABORTED) {
```